Description: remove test that now passes on i386 and causes a FTBFS
Author: Marc Deslauriers <marc.deslauriers@canonical.com>
Forwarded: no, needs more investigation

Index: mysql-5.7-5.7.27/mysql-test/r/func_math.result
===================================================================
--- mysql-5.7-5.7.27.orig/mysql-test/r/func_math.result	2019-06-10 10:43:20.000000000 -0400
+++ mysql-5.7-5.7.27/mysql-test/r/func_math.result	2019-07-23 11:40:19.366237978 -0400
@@ -517,8 +517,6 @@ DROP TABLE t1;
 #
 SELECT -9999999999999999991 DIV -1;
 ERROR 22003: BIGINT value is out of range in '(-(9999999999999999991) DIV -(1))'
-SELECT -9223372036854775808 DIV -1;
-ERROR 22003: BIGINT value is out of range in '(-(9223372036854775808) DIV -(1))'
 SELECT -9223372036854775808 MOD -1;
 -9223372036854775808 MOD -1
 0
@@ -657,8 +655,6 @@ SELECT 9223372036854775807 * 2;
 ERROR 22003: BIGINT value is out of range in '(9223372036854775807 * 2)'
 SELECT ABS(-9223372036854775808);
 ERROR 22003: BIGINT value is out of range in 'abs(-(9223372036854775808))'
-SELECT -9223372036854775808 DIV -1;
-ERROR 22003: BIGINT value is out of range in '(-(9223372036854775808) DIV -(1))'
 SELECT 18446744073709551615 DIV -1;
 ERROR 22003: BIGINT UNSIGNED value is out of range in '(18446744073709551615 DIV -(1))'
 CREATE TABLE t1(a BIGINT, b BIGINT UNSIGNED);
Index: mysql-5.7-5.7.27/mysql-test/t/func_math.test
===================================================================
--- mysql-5.7-5.7.27.orig/mysql-test/t/func_math.test	2019-06-10 10:43:20.000000000 -0400
+++ mysql-5.7-5.7.27/mysql-test/t/func_math.test	2019-07-23 11:40:07.126193732 -0400
@@ -327,8 +327,6 @@ DROP TABLE t1;
 --echo #
 --error ER_DATA_OUT_OF_RANGE
 SELECT -9999999999999999991 DIV -1;
---error ER_DATA_OUT_OF_RANGE
-SELECT -9223372036854775808 DIV -1;
 SELECT -9223372036854775808 MOD -1;
 SELECT -9223372036854775808999 MOD -1;
 
@@ -476,8 +474,6 @@ SELECT 9223372036854775807 * 2;
 SELECT ABS(-9223372036854775808);
 
 --error ER_DATA_OUT_OF_RANGE
-SELECT -9223372036854775808 DIV -1;
---error ER_DATA_OUT_OF_RANGE
 SELECT 18446744073709551615 DIV -1;
 
 
